src/app/data/components/data-browser/data-browser.component.html

```diff
@@ -969,8 +974,10 @@ <h1 class="text-lg text-gray-900 mb-2 text-center">Info</h1>
                     {{item.value.substring(item.value.startsWith("https")?8:7)}}</span>
 
             </div>
-            <div>
-                <button (click)="testLink(item.value + '?pos=1&type=DATA_SLICE')"
+            <div [attr.data-tip]="!isManaged ? 'Only usable for the managed version' : ''"
```

Comment:
doesn't need to be bound since having it without the tooltip class doesn't do anything :)